rackcity/views/it_instance_views.py

```diff
@@ -368,24 +391,6 @@ def instance_bulk_upload(request):
                 )
             else:
                 instances_to_add.append(instance_serializer)
-        else:
-            try:
-                validate_location_modification(
-                    instance_data, existing_instance)
-            except Exception:
-                failure_message = "Instance " + \
-                    instance_data['hostname'] + \
-                    " would conflict location with an existing instance. "
-                return JsonResponse(
-                    {"failure_message": failure_message},
-                    status=HTTPStatus.BAD_REQUEST
-                )
-            potential_modifications.append(
-                {
-                    "existing_instance": existing_instance,
-                    "new_data": instance_data
-                }
-            )
```

Comment:
moved above since this was `try...except...else` and now it's just `if...else...`